tmt-app/src/Components/MajorPage.jsx

```diff
@@ -1,10 +1,38 @@
-import React from "react";
+import { Container, Typography } from "@mui/material";
+import React, { useEffect, useState } from "react";
 import { useParams } from "react-router-dom";
+import { MAJORS_LIST } from "../constants/careerMajorList";
 
 function MajorPage() {
-  const majorChoice = useParams();
+  const params = useParams();
+  const [data, setData] = useState([]);
 
-  return <div>{majorChoice.major}</div>;
+  const searchData = (target) => {
+    for (let i = 0; i < MAJORS_LIST.length; i++) {
+      if (target === MAJORS_LIST[i].key) {
+        setData(MAJORS_LIST[i]);
+        console.log(MAJORS_LIST[i]);
+      }
+    }
+  };
+
+  useEffect(() => {
+    searchData(params.major);
+  }, [params]);
+
+  return (
+    <Container>
+      <Typography
+        variant="h1"
+        sx={{
+          margin: "0 auto",
```

Comment:
Margin 0 is pushing the title up against the top of the page with zero space, is this is necessary style to develop against the designs?

Reply:
The title does not account for breadcrumb and nav bar yet. This would also help center the text but I could remove it since I already have textAlign to do that.

Reply:
Ah I see, if we've got some inherited margin, i dont think we need to remove it now until we get the nav bar and breadcrumbs in, we may end up just undoing this if we find that we need it later